### PR TITLE
deployment flow sections

### PR DESCRIPTION
**Pull request by huwshimi**
Add a component for handling displaying and disabling sections etc.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- use the df flag
- load the deployment flow
- it should look like it did in my previous branch

File: jujugui/static/gui/src/app/components/deployment-flow/section/section.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      @method _generateCheck
+      @returns {Object} The actions markup.
+    */
+    _generateActions: function() {
+      var buttons = this.props.buttons;
+      if (!buttons) {
+        return;
+      }
+      return (
+        <div className="deployment-section__actions">
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/section/section.js]: Button row already wraps itself in a div, maybe we should add the ability to pass in custom classes for it

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5427/
Test PASSed.

**Comment by hatched** [comment]
Looks good thanks @huwshimi  :+1: :ok:

**Comment by huwshimi** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**